pkg/errutil/errors_check.go

```diff
@@ -33,3 +33,28 @@ func IsErrConnectionRefused(err error) bool {
 	const errMessage = "connect: connection refused"
 	return strings.Contains(err.Error(), errMessage)
 }
+
+// IsExactArgs returns an error if there is not the exact number of args
+func IsExactArgs(number int) cobra.PositionalArgs {
```

Comment:
[`cmd/nerdctl/main.go`](https://github.com/containerd/nerdctl/blob/68abe45c72c250fe9666ae6c23ab16ff81bff127/cmd/nerdctl/main.go) might be a better place for this function. It contains most cobra related functions.